torch/export/exported_program.py

```diff
@@ -411,18 +413,22 @@ def _decompose_and_get_gm_with_new_signature_constants(
                                     entry
                                 ]
 
-        aten_export_artifact = _export_to_aten_ir(
-            mod,
-            # this requires empty kwargs, but not in pytree.flattened format
-            (
-                *fake_args_unwrapped[0],
-                *fake_args_unwrapped[1].values(),
-            ),
-            {},
-            fake_params_buffers,
-            constant_attrs,
-            _check_autograd_state=False,
-        )
+        with _ignore_backend_decomps(), fake_mode, _override_composite_implicit_decomp(
```

Comment:
this is also now duped with the non-training path, I think?

Reply:
Hmm there is a bit of problem here. Even though the context manager is the same, we call different mechanism to get the aten IR. In the training IR case, we actually need sort of "retracing" kind of way (hence export_to_aten_ir), but in the non-training path, we call aot_aexport_module directly. I am still thinking about how to refactor this properly tho.